### ISSUE TITLE
Report Scenario Outlines as tables

### ISSUE THREAD

**Issue by Chirag-SSENSE** [serenity-js/serenity-js#168]
I am trying to test few users using the feature file, this is what my feature file looks like:

//example.feature
`Feature: Add item and place the order for that item
	In order to wear something new
	As an online shopping customer
	I want to be able to add an item to my shopping bag and then place the order

	Scenario Outline: Add an item to shopping bag to place the order using PaypalPro
		Given "<User>" is logged in to his "<EnvName>" account
		 When he searches item with SKU "<SKUID>" from "<Gender>" section and adds it to his shopping bag
		 Then "<User>" can place the order
	Examples:
		| User         | EnvName | SKUID        | Gender |
		| CanadaQCUser | SSENSE  | 51230M000010 | men    |
		| GBUser       | SSENSE  | 51230M000010 | women  |
		| FranceUser   | SSENSE  | 51230M000010 | kids   |
		| USUser       | SSENSE  | 51230M000010 | men    |`

When I run these tests the report only generate results for the last user, as you can see in this case it only generated for the USUser in this case

<img width="1204" alt="screen shot 2018-02-14 at 11 15 36 am" src="https://user-images.githubusercontent.com/35231678/36214834-6f6d5d98-1178-11e8-8135-b168f13e7933.png">

**Comment by jan-molak** [comment]
Hi @Chirag-SSENSE, yes, it is a limitation of the current cucumber-js scenario outline integration, but I'm working to resolve it.
As a workaround you can modify the scenario title to include something that would distinguish one scenario from another, for example:

```gherkin
Scenario Outline: Add an item to shopping bag to place the order using PaypalPro (<User>)
	Given "<User>" is logged in to his "<EnvName>" account
	 When he searches item with SKU "<SKUID>" from "<Gender>" section and adds it to his shopping bag
	 Then "<User>" can place the order
Examples:
	| User         | EnvName | SKUID        | Gender |
	| CanadaQCUser | SSENSE  | 51230M000010 | men    |
	| GBUser       | SSENSE  | 51230M000010 | women  |
	| FranceUser   | SSENSE  | 51230M000010 | kids   |
	| USUser       | SSENSE  | 51230M000010 | men    |
```

**Comment by Chirag-SSENSE** [comment]
@jan-molak , can you please give me an example what do you mean?

**Comment by Chirag-SSENSE** [comment]
@jan-molak , is it possible to get a rough estimation when it can be fixed? The workaround will be messy for us because we have a lot of examples.

**Comment by jan-molak** [comment]
Hi @Chirag-SSENSE - I've just started looking into this, so depending on the amount of free time I get it can be anything between a week and a couple of weeks. Sorry, but client work takes a priority.

The simplest workaround you could use in the meantime is to add an `id` column to each example and then use that to distinguish between them:

```
Scenario Outline: Add an item to shopping bag to place the order using PaypalPro (<User>)
    Given "<User>" is logged in to his "<EnvName>" account
     When he searches item with SKU "<SKUID>" from "<Gender>" section and adds it to his shopping bag
     Then "<User>" can place the order

Examples:
	| Id | User         | EnvName | SKUID        | Gender |
	| 1  | CanadaQCUser | SSENSE  | 51230M000010 | men    |
	| 2  | GBUser       | SSENSE  | 51230M000010 | women  |
	| 3  | FranceUser   | SSENSE  | 51230M000010 | kids   |
	|4  | USUser       | SSENSE  | 51230M000010 | men    |
```

Does that help for now?

**Comment by AbhineetSharmax** [comment]
Hi @jan-molak,

Greetings!!

Have faced the similar thing with scenario outline examples, do we have a workaround for it? please suggest.

Thanks :)

**Comment by expphchen** [comment]
I am researching solution on this on cucumberJS json formatter, I found it maybe useful rewrite
https://github.com/rapid7/cucumber-json_expanded/blob/master/lib/cucumber/json_expanded/formatter.rb
for the JS version. Any volunteer?

**Comment by jan-molak** [comment]
This will be solved properly in Serenity/JS 2.0, so we'll be able to report on scenarios such as [those here](https://github.com/jan-molak/serenity-js/blob/2.0/packages/cucumber/spec/gherkin/feature-files/fixtures/scenario_outlines.feature) :-) Please stay tuned!

**Comment by AbhineetSharmax** [comment]
Kudos!! 🍀
